contracts/test/MapleGlobals.t.sol

```diff
@@ -1,273 +1,439 @@
+        assertTrue( realGov.try_mapleGlobals_setLpCooldownPeriod(address(globals), 1 days));
+        assertEq(globals.lpCooldownPeriod(), 1 days);
+    }
+    
+    function test_setStakerUnstakeWindow() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
 
-        assertTrue(globals.isValidSubFactory(address(poolFactory), address(slFactory),  SL_FACTORY));
-        assertTrue(globals.isValidSubFactory(address(poolFactory), address(llFactory),  LL_FACTORY));
-        assertTrue(globals.isValidSubFactory(address(poolFactory), address(dlFactory1), DL_FACTORY));
-        assertTrue(globals.isValidSubFactory(address(loanFactory), address(clFactory),  CL_FACTORY));
-        assertTrue(globals.isValidSubFactory(address(loanFactory), address(flFactory),  FL_FACTORY));
+        assertTrue(!fakeGov.try_mapleGlobals_setStakerUnstakeWindow(address(globals), 1 days));
+        assertTrue( realGov.try_mapleGlobals_setStakerUnstakeWindow(address(globals), 1 days));
+        assertEq(globals.stakerUnstakeWindow(), 1 days);
     }
+    
+    function test_setLpWithdrawWindow() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertTrue(!fakeGov.try_mapleGlobals_setLpWithdrawWindow(address(globals), 1 days));
+        assertTrue( realGov.try_mapleGlobals_setLpWithdrawWindow(address(globals), 1 days));
+        assertEq(globals.lpWithdrawWindow(), 1 days);
+    }
+    
+    function test_setMaxSwapSlippage() public {
+        (Governor realGov, Governor fakeGov, MapleGlobals globals) = makeGlobalsAndGovs();
+
+        assertTrue(!fakeGov.try_mapleGlobals_setMaxSwapSlippage(address(globals), 10_000));
+        assertTrue(!realGov.try_mapleGlobals_setMaxSwapSlippage(address(globals), 10_001));  // Out of range
+        assertTrue( realGov.try_mapleGlobals_setMaxSwapSlippage(address(globals), 10_000));
+        assertEq(globals.maxSwapSlippage(), 10_000);
+    }
+    
+    function test_setGlobalAdmin() public {
+        Governor     realGov         = new Governor();
+        Governor     fakeGov         = new Governor();
+        GlobalAdmin  realGlobalAdmin = new GlobalAdmin();
+        MapleGlobals globals         = new MapleGlobals(address(realGov), address(1), address(realGlobalAdmin));
+
+        assertTrue(!fakeGov.try_mapleGlobals_setGlobalAdmin(address(globals), address(1)));
+        assertTrue(!realGov.try_mapleGlobals_setGlobalAdmin(address(globals), address(0)));  // Can't set to zero address
+
+        realGlobalAdmin.mapleGlobals_setProtocolPause(address(globals), true);
+        assertTrue(!realGov.try_mapleGlobals_setGlobalAdmin(address(globals), address(1)));
+        
+        realGlobalAdmin.mapleGlobals_setProtocolPause(address(globals), false);
+        assertTrue( realGov.try_mapleGlobals_setGlobalAdmin(address(globals), address(1)));
+    }
```

Comment:
`makeGlobalsAndGovs` not used again. Perhaps throw this in the other contract and `MapleGlobalsGeneralTests`.